ui/app/common/events/events-services.js

```diff
@@ -92,14 +92,18 @@
                 payload: event.payload
               };
 
-              switch (event.payload.buildStatus) {
-                case 'REPO_SETTING_UP':
+              switch (event.payload.buildCoordinationStatus) {
```

Comment:
@matejonnet Matej, any status which you think we should ignore in UI? All these will issue a Notification ("The build... has started") or ("The build xx completed, failed, etc)